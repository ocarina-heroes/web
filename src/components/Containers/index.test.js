import React from 'react'
import renderer from 'react-test-renderer'
import 'jest-styled-components'
import { Root, Main } from '.'

test('<Root/>> matches snapshot', () => {
  const tree = renderer.create(<Root />).toJSON()
  expect(tree).toMatchSnapshot()
})

test('<Main/>> matches snapshot', () => {
  const tree = renderer.create(<Main />).toJSON()
  expect(tree).toMatchSnapshot()
})
