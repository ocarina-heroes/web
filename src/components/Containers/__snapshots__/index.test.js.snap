// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main/>> matches snapshot 1`] = `
.c0 {
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

<div
  className="c0"
/>
`;

exports[`<Root/>> matches snapshot 1`] = `
.c0 {
  min-height: 100vh;
  padding: 0 0.5rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

<div
  className="c0"
/>
`;
